{{cookiecutter.projectDirectory}}/{{cookiecutter.projectName}}/Code/SceneDelegate.swift

```diff
@@ -18,23 +17,22 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window.makeKeyAndVisible()
     }
 
-    func sceneDidDisconnect(_: UIScene) {
-    }
+    func sceneDidDisconnect(_: UIScene) {}
 
-    func sceneDidBecomeActive(_: UIScene) {
-    }
+    func sceneDidBecomeActive(_: UIScene) {}
 
-    func sceneWillResignActive(_: UIScene) {
-    }
+    func sceneWillResignActive(_: UIScene) {}
 
     func sceneWillEnterForeground(_: UIScene) {
         guard AppEnvironment.current.buildConfig != .debug else { return }
 
         Task {
+            await ForceUpdateController.shared.configure(
```

Comment:
das configure muss hier nicht nochmal passieren, das checkForUpdates reicht in diesem fall